cogs/assistance-cmds/redownloadcfw.wiiu.md

```diff
@@ -8,5 +8,7 @@ If homebrew is behaving strangely, redownloading your files should help.
 **Tiramisu:**
 https://tiramisu.foryour.cafe/
 
+**OR**
+
 **Aroma:**
 https://aroma.foryour.cafe/
```

Comment:
Not a blocker, but given Aroma is main now, switch the order around?

(Do we still need Tiramisu here even?)

Reply:
I suppose I could do that, but yes Maschell said people can keep using Tiramisu if they want, for legacy purposes

Reply:
Not privy to Wii U things, so if it's good to leave it then it's fine. But Aroma should still be at the top of the list since it is considered the "default" for our server now.